Automattic/jetpack
# Podcast Player: fix invisible controls issue

#### Changes proposed in this Pull Request:

This PR aims to fix an issue where the Podcast Player controls are invisible when WordPress is installed in a sub-folder. For further details and related discussion please see issue https://github.com/Automattic/jetpack/issues/15611.

Fixes https://github.com/Automattic/jetpack/issues/15611

#### Testing instructions:






~~Note: currently there is a bug in master that causes a type error for the podcast player. To test please apply the fix in https://github.com/Automattic/jetpack/pull/15854 while that PR isn't merged yet.~~

* Have WordPress installed in a sub-folder
* Add the Podcast Player to a post, add a podcast (e.g. anchor.fm/dissect) and confirm the icons are now showing as expected
* Use a non-subfolder WordPress installation 
* Add the Podcast Player to a post, add a podcast (e.g. anchor.fm/dissect) and confirm the icons are still showing as expected

#### Proposed changelog entry for your changes:

* Restores podcast player icons for subfolder installations and sites using Photon.
